### PR TITLE
Remove obsolete providers

### PR DESCRIPTION
**Pull request by j-xiong**

### CODE REVIEW

**Comment by j-xiong** [comment]
@shijin-aws I don't think so. The API doesn't change.

**Comment by shijin-aws** [comment]
Thank you. AWS still needs a 1.21 release for a new feature https://github.com/ofiwg/libfabric/pull/9525 that is being development in the current main branch. IIUC you said it should be still be ok to cut 1.x release from main before the (3) happens?

**Review by soumagne — APPROVED** [review]

**Comment by garzaran** [comment]
Got it

Maria


On Dec 8, 2023, at 1:48 PM, Jianxin Xiong ***@***.******@***.***>> wrote:



@ofiwg/intel-middleware<https://github.com/orgs/ofiwg/teams/intel-middleware>

File: .travis.yml [hunk 1]
```diff
@@ -87,22 +87,18 @@ install:
     # Test loadable library option
     # List of providers current as of Jan 2020
     - ./configure --prefix=$PREFIX --enable-tcp=dl
-      --disable-bgq
       --disable-efa
-      --disable-gni
       --disable-hook_debug
       --disable-mrail
       --disable-perf
       --disable-psm
```
Comment (acgoldma) [inline .travis.yml:94]: psm (v1) should have already be removed. so you should be able to remove `--disable-psm`

**Comment by jswaro** [comment]
Why is GNI being removed?

**Comment by j-xiong** [comment]
@jswaro The GNI provider is not being actively maintained (the last update was 2 years ago). Please feel free to provide your input here.

**Comment by jswaro** [comment]
> @jswaro The GNI provider is not being actively maintained (the last update was 2 years ago). Please feel free to provide your input here.

I'll ask around. I don't know if anyone is directly impacted. I haven't received an answer yet.

**Pull request merged**